tests/node/node.go

```diff
@@ -24,11 +24,11 @@ import (
 	"os/exec"
 	"strings"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	test "github.com/onsi/ginkgo"
+	matcher "github.com/onsi/gomega"
```

Comment:
Why are we changing this? The dot import is widely practiced in Go projects I saw. And we leave it in other files here (ovs_test.go).

Reply:
I've renamed the node_test.go into node.go (because it contains no tests) as there were CI failures in an earlier commit.
when go module file name is not with *_test.go, golint throws error for dot imports. 

But it looks like CI failure was due to clusterApi referencing issue in the test suite. 
now changed it back to node_test.go and cluser_test.go. Hope CI succeeds now.